homepage/templates/navbar.html

```diff
@@ -1,18 +1,21 @@
 <nav class="navbar navbar-expand-sm navbar-dark">
-	<a class="navbar-brand" href="#">مای‌آکادمی</a>
```

Comment:
Please convert space indentations to tabs.

Reply:
Done. Check again just incase